metaseq/utils.py

```diff
@@ -93,6 +124,10 @@ def make_positions(tensor, padding_idx: int):
     return (torch.cumsum(mask, dim=1).type_as(mask) * mask).long() + padding_idx
 
 
+def strip_pad(tensor, pad):
```

Comment:
thought: this could be renamed to `strip_token`, since there's no real constraing that `pad==<PAD>`